src/components/search/search.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { screen, render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { SearchView } from './search';
+
+const noop = jest.fn();
+
+describe('Search', () => {
+  test('renders SearchSnapshot', () => {
+    const { asFragment } = render(<SearchView onSearch={noop} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should call on search with userText', () => {
+    const onSearch = jest.fn();
```

Comment:
In general, I definitely advice to follow AAA pattern for UTs. This approach makes test easier to read and setup. Take a look at more examples [here](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80)